quick_start.adoc

```diff
@@ -261,17 +261,20 @@ An entry under `identityProviders` has the following fields:
   * `baseDN` contains information to start the LDAP user search from. For example "cn=users,dc=example,dc=com"
   * `filter` contains optional filter to apply when searching the directory. For example "(objectClass=person)"
 - `ldap` contains the LDAP specific configurations:
-  * `url` contains the LDAP host and optional port of the LDAP server
+  * `url` contains the LDAP host and search parameters to use. The url syntax is `ldap://host:port/basedn?attribute?scope?filter`
+  * `insecure` must be `false`. ldaps:// URLs will attempt to connect using TLS, and ldap:// URLs will be upgraded to TLS. Identity Co Gnfiguration Management Operator does not support insecure=true. 
   * `bindDN` contains an optional DN to bind with during the search phase.
   * `bindPassword` contains an optional reference to a secret by name containing a password to bind with during the search phase.
 *Note*: bindDN and bindPassword need to be provided if the LDAP server requires authentication for search.
   * `ca` contains reference to the secret containing a trusted Root CA file - file name and format: "ca.crt"
 *Note*: If the server uses self-signed certificates, include files with names "tls.crt" and "tls.key" (representing client certificate and key) in the same secret
   - `attributes` maps LDAP attributes to identities
-    * `id` is the list of attributes whose values should be used as the user ID. Required. First non-empty attribute is used. At least one attribute is required. If none of the liste attribute have a value, authentication fails. LDAP standard identity attribute is "dn"
-    * `preferredUsername` is the list of attributes whose values should be used as the preferred username. LDAP standard login attribute is "uid"
-    * `name` is the list of attributes whose values should be used as the display name. Optional. If unspecified, no display name is set for the identity. LDAP standard display name attribute is "cn"
-    * `email` is the list of attributes whose values should be used as the email address. Optional. If unspecified, no email is set for the identity
+    * `id` is the attribute whose values should be used as the user ID. Required. First attribute is used. At least one attribute is required. If none of the listed attribute have a value, authentication fails. LDAP standard identity attribute is "dn". 
+    * `preferredUsername` is the attributes whose values should be used as the preferred username. LDAP standard login attribute is "uid". Identity Configuration Management Operator ignore this value. 
```

Comment:
I think we can leave off the description of what it is since we're ignoring it and just say it's ignored. Please use lowercase on "Identity Configuration Management Operator" for consistency with the rest of the docs. The official name is lowercase. I'm not sure what the "LDAP standard login attribute is "uid"." is referring to exactly. Do we need to capture this point elsewhere or add some more clarification?